consensus/state.go

```diff
@@ -1335,6 +1335,8 @@ func (cs *State) defaultDecideProposal(height int64, round int32) {
 
 		cs.propagator.ProposeBlock(proposal, blockParts, metaData)
 
+		cs.sendInternalMessage(msgInfo{&ProposalMessage{proposal}, ""})
```

Comment:
why is this being re-sent? The proposal seems to already be sent on line 1324

Reply:
damn, added this originally and I could swear it fixed the e2e test but its definitely duplicated so I guess not...